src/views/distribution-lists-view.test.tsx

```diff
@@ -251,7 +251,8 @@ describe('Distribution Lists View', () => {
 										owners: resData.owners?.map((owner) => ({ owner: [owner] })),
 										_attrs: {
 											displayName: resData.displayName,
-											description: resData.description
+											description: resData.description,
+											zimbraHideInGal: 'FALSE'
```

Comment:
By updating the type to set the field as optional, this line is not required anymore